### DISCUSSION TITLE
When is the Gantt chart going to be available in the library ?

### DISCUSSION THREAD

**Discussion by Hervera** [carbon-design-system/carbon-charts#1794] [General]
Hi, Do you have any plans of having a Gantt Chart in the library ?

**Comment by vgallegob** [comment]
That would be awesome

**Comment by theiliad** [comment]
I don't think that would be the right approach to ship Gantt charts as part of this library. Gantt charts wouldn't be easily configurable through a set of data & options.

I would think a React component library would work instead

However, we don't have any plans to build gantt chart components
